Rakefile

```diff
@@ -12,4 +12,16 @@ Rake::TestTask.new do |t|
   t.warning = false
 end
 
+# For code coverage measurements to work properly, `SimpleCov` should be loaded
+# and started before any application code is loaded.
+if ENV["COVERAGE"]
+  require "simplecov"
+  require "simplecov_json_formatter"
+  SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([SimpleCov::Formatter::HTMLFormatter, SimpleCov::Formatter::JSONFormatter])
+  SimpleCov.command_name "rspec-unit"
+  SimpleCov.start do
+    track_files "test/**/*.{rb}"
+  end
+end
```

Comment:
`SimpleCov.start` should be run from `test_helper.rb` instead of `Rakefile`. Then it'll work.